.eslintrc.json

```diff
@@ -476,6 +477,7 @@
 					"azdata",
 					"mssql",
 					"azurecore",
+					"mssql",
```

Comment:
All of these are duplicated - mssql is already in the lists for each of these sections.